slash/general.py

```diff
@@ -32,6 +32,59 @@ async def github(self, interaction: Interaction):
             embed=embed, view=Link("https://github.com/codeman1o1/TIJK-Bot")
         )
 
+    @slash(guild_ids=SLASH_GUILDS)
+    async def avatar(
+        self,
+        interaction: Interaction,
+        user: nextcord.Member = SlashOption(
+            description="The user of which you wish to download their pfp",
```

Comment:
This would be better
```suggestion
            description="The user of which you want to download their avatar",
```